rsconf/component/mongod.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+"""create nginx configuration
+
+:copyright: Copyright (c) 2017 RadiaSoft LLC.  All Rights Reserved.
+:license: http://www.apache.org/licenses/LICENSE-2.0.html
+"""
+from pykern.pkcollections import PKDict
+from pykern.pkdebug import pkdp
+from rsconf import component
+
+_DATA_DIR = "db"
+
+_MONGO_CONF_F = "/etc/mongod.conf"
```

Comment:
It would be ideal to move this /srv/mongod

Reply:
6c04a19